spec/dummy/app/models/application_record.rb

```diff
@@ -0,0 +1,5 @@
+if Rails::VERSION::STRING >= '5.2.0'
+  class ApplicationRecord < ActiveRecord::Base
```

Comment:
Models generated on Rails 5.2 (and edge) were expecting an `ApplicationRecord` to have been defined in the dummy app.